src/button/high-visibility.scss

```diff
@@ -31,4 +30,10 @@
       }
     }
   }
+
+  &:focus {
+    @include themed {
+      outline-color: t(iui-color-foreground-accessory);
```

Comment:
@FlyersPh9 What do you think of only changing (in #166) the outline-color and leaving the offset as -1px for hv/cta buttons? Looks more consistent imo. But we could play with the opacity ofc
![image](https://user-images.githubusercontent.com/9084735/139932286-70157318-bc63-486e-a719-cf7af84cf7bb.png)
![image](https://user-images.githubusercontent.com/9084735/139932461-551ebbd0-4edd-44e2-8f43-1663026fd1db.png)

Reply:
ah wait, it is not visible in light theme so we'd need a different color

Reply:
1 option would be to use the same color as body text, but I think a black border on a blue button in light theme won't look so great.  Maybe if we combine the body text color with opacity so some of the blue / green button comes through, maybe that will look better?

Reply:
I reverted the focus styling for now. Lets handle it in yours.